[2025-09-02T07:39:13.161+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-09-02T07:39:13.207+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: pipeline_data_rekapitulasi.load_to_postgres manual__2025-09-02T07:38:53.702175+00:00 [queued]>
[2025-09-02T07:39:13.223+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: pipeline_data_rekapitulasi.load_to_postgres manual__2025-09-02T07:38:53.702175+00:00 [queued]>
[2025-09-02T07:39:13.225+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2025-09-02T07:39:13.258+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): load_to_postgres> on 2025-09-02 07:38:53.702175+00:00
[2025-09-02T07:39:13.288+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=501) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-09-02T07:39:13.296+0000] {standard_task_runner.py:63} INFO - Started process 503 to run task
[2025-09-02T07:39:13.297+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'pipeline_data_rekapitulasi', 'load_to_postgres', 'manual__2025-09-02T07:38:53.702175+00:00', '--job-id', '5', '--raw', '--subdir', 'DAGS_FOLDER/pipeline_rekapitulasi.py', '--cfg-path', '/tmp/tmp0mk2xhdh']
[2025-09-02T07:39:13.302+0000] {standard_task_runner.py:91} INFO - Job 5: Subtask load_to_postgres
[2025-09-02T07:39:13.449+0000] {task_command.py:426} INFO - Running <TaskInstance: pipeline_data_rekapitulasi.load_to_postgres manual__2025-09-02T07:38:53.702175+00:00 [running]> on host 231ead08851d
[2025-09-02T07:39:13.677+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='pipeline_data_rekapitulasi' AIRFLOW_CTX_TASK_ID='load_to_postgres' AIRFLOW_CTX_EXECUTION_DATE='2025-09-02T07:38:53.702175+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-09-02T07:38:53.702175+00:00'
[2025-09-02T07:39:13.682+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-09-02T07:39:13.734+0000] {base.py:84} INFO - Using connection ID 'postgres_db_conn' for task execution.
[2025-09-02T07:39:13.775+0000] {logging_mixin.py:188} INFO - Table siap
[2025-09-02T07:39:13.881+0000] {logging_mixin.py:188} INFO - Data berhasil dimasukkan ke Postgres: 31 baris
[2025-09-02T07:39:13.882+0000] {python.py:237} INFO - Done. Returned value was: None
[2025-09-02T07:39:13.883+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-09-02T07:39:13.901+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=pipeline_data_rekapitulasi, task_id=load_to_postgres, run_id=manual__2025-09-02T07:38:53.702175+00:00, execution_date=20250902T073853, start_date=20250902T073913, end_date=20250902T073913
[2025-09-02T07:39:13.958+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-09-02T07:39:14.000+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-09-02T07:39:14.004+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
